code/game/gamemodes/changeling/modularchangling.dm

```diff
@@ -65,7 +65,7 @@ var/list/datum/power/changeling/powerinstances = list()
 
 /datum/power/changeling/horror_form
 	name = "Horror Form"
-	desc = "This costly evolution allows us to transform into an all-consuming abomination. We are extremely strong, to the point that we can force airlocks open and devour humans whole, and immune to stuns."
+	desc = "This costly evolution allows us to transform into an all-consuming abomination. We are extremely strong, to the point that we can force airlocks, and immune to stuns."
```

Comment:
"We are incredibly strong, to the point that we can force open airlocks, and are immune to conventional stuns."